backend/classification/config/constants.py

```diff
@@ -8,13 +8,22 @@ class Sex(Enum):
     M = 2
 
 
+class AcquisitionBoard(Enum):
+    OPENBCI_CYTON = 1
+    OPENBCI_GANGLION = 2
```

Comment:
```suggestion
    Cyton = auto()
    Ganglion = auto()
```

Afin d'avoir des noms human friendly?

Reply:
On va probablement devoir changer la query http